windows/src/desktop/help/index.md

```diff
@@ -2,16 +2,51 @@
 title: Keyman 14.0 Help
 ---
 
-* [About Keyman](about/)
+Need help using Keyman for Windows? You'll find everything you need here, including product documentation,
+frequently asked questions, tutorials, and videos.
 
-* [Getting Started](start/)
+## [Getting Started](start/tutorial)
+* [Step-by-step Guide](start/tutorial)
+* [Installing a keyboard](start/download-and-install-keyboard)
 
-* [Basic Help](basic/)
+## [Common Questions](common/)
+* [Which operating systems does Keyman for Windows work with?](common/requirements)
+* [Does Keyman for Windows run on Mac?](common/mac)
 
-* [Advanced Help](advanced/)
+## Ask for Help
+[Ask other users in the SIL Keyman Keyman Community](https://community.software.sil.org/c/keyman)
 
-* [Common Questions](common/)
+## Help Topics
 
-* [Troubleshooting](troubleshooting/)
+### [Troubleshooting](troubleshooting/)
+* [Square boxes or fonts not showing correctly](/kb/42)
```

Comment:
I didn't know if these should be full paths for embedded help. (To production help.keyman.com?)

Reply:
Yes, should be full paths to production.